test/run-tests

```diff
@@ -443,7 +443,7 @@ def get_comment_commands(gh, owner, repo, pullnr):
     return commands
 
 
-def choose_task(gh, repos, images):
+def choose_task(gh, repos, images, config):
```

Comment:
In general IMHO the architecture should be changed to for example a `TestHarness` class which has properties like the config, so that the methods like `choose_task()` could just access them via `self.config`.